### PR TITLE
Add new tabs for the task properties to be coherent as workflow properties

### PR DESCRIPTION
**Pull request by tao-xinxiu**
Add new tabs 'Task Variables' and 'Generic Info' for the task properties to be coherent as workflow properties.

### CODE REVIEW

File: app/scripts/proactive/model/Task.js [hunk 1]
```diff
@@ -58,13 +58,21 @@ define(
...
+                        "data-help": 'Indicates if you want to save the result of this task in the job result.'
+                    }
+                },
                 "Variables": {
                     type: 'List',
                     itemType: 'Object',
                     title: "Task Variables (Name, Value, Inherited)",
                     fieldAttrs: {
                         'placeholder': 'variables->variable',
+                        "data-tab": "Task Variables",
```
Comment (fviale) [inline app/scripts/proactive/model/Task.js:74]: why not Variables ?

Comment (tao-xinxiu) [reply app/scripts/proactive/model/Task.js:74]: Here is the tab name, I feel it might be better to clarify it is just the "Task Variables"?

**Comment by tao-xinxiu** [comment]
retest this please

File: app/scripts/proactive/model/Task.js (hunk 1 above)
Comment (fviale) [reply app/scripts/proactive/model/Task.js:74]: maybe you should ask d.c. before merging it ?

File: app/scripts/proactive/model/Task.js [hunk 2]
```diff
@@ -84,7 +92,8 @@ define(
                     itemType: 'Object',
                     fieldAttrs: {
                         'placeholder': 'genericInformation->info',
-                        "data-help": 'Some extra information about your job often used to change the scheduling behavior for a job. E.g. NODE_ACCESS_TOKEN=rack1 will assign this task to a node with token \"rack1\".'
+                        "data-tab": "Generic Info",
```
Comment (mboussaa) [inline app/scripts/proactive/model/Task.js:95]: Task Generic Info 
or Generic Task Info 
To be checked with d.c as well

Comment (mboussaa) [reply app/scripts/proactive/model/Task.js:95]: you can show the screenshot in Trello and tag him to have his pov of both remarks

**Review by mboussaa — APPROVED** [review]

**Pull request merged**